package.json

```diff
@@ -1,37 +1,68 @@
 {
-  "name": "load-secrets-action",
-  "version": "1.1.0",
-  "description": "Load Secrets from 1Password",
-  "main": "dist/index.js",
-  "directories": {
-    "test": "tests"
-  },
-  "scripts": {
-    "build": "ncc build src/index.ts"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/1Password/load-secrets-action.git"
-  },
-  "keywords": [
-    "actions",
-    "1password",
-    "load secrets",
-    "connect"
-  ],
-  "author": "1Password",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/1Password/load-secrets-action/issues"
-  },
-  "homepage": "https://github.com/1Password/load-secrets-action#readme",
-  "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.1"
-  },
-  "devDependencies": {
-    "@types/node": "^18.14.2",
-    "@vercel/ncc": "^0.36.1",
-    "typescript": "^4.9.5"
-  }
+	"name": "load-secrets-action",
+	"version": "1.1.0",
```

Comment:
Since we're doing some tooling improvement here, would it be ok to include it in this PR making this version match the action's version? (currently `1.2.0`)

Reply:
I can bump this manually to `1.2.0` if it makes sense to do so 👍

Seeing as how we already have a mismatch between the `package.json` version and the released version, IMO we should consider automating this using something like [semantic-release](https://github.com/semantic-release/semantic-release), a tool designed to be used in CI/CD to automatically manage versioning.

Reply:
Using an automation tool sounds like an awesome idea. However, I'd suggest investigating and providing such solution in a separate PR if that makes more sense. 

Taking a quick look at the [semantic-release](https://github.com/semantic-release/semantic-release) tool, it looks like there may be more to do on our side than just using it (e.g. commit format).

Therefore, here's my suggestion for this:
- Manually bump the version in this PR to no longer have the mismatch.
- Later add a tool to automatically manage versioning so that such mismatches no longer happen.